source/examples/demo-stages/include/demo-stages/ColorizeStage.h

```diff
@@ -27,7 +29,7 @@ namespace gloperate
 *  @brief
 *    Demo stage that applies a color to a texture
 */
-class GLOPERATE_API ColorizeStage : public Stage
+class DEMO_STAGES_API ColorizeStage : public Stage
```

Comment:
Demo stages don't need a public API.
@sbusch42 or do we want to enable for both plugin usage and dynamic lib usage for these example stages?